mindmeld/components/_config.py

```diff
@@ -928,6 +937,37 @@ def get_nlp_config(app_path=None, config=None):
     return _get_default_nlp_config()
 
 
+def get_auto_annotator_config(app_path=None, config=None):
+    """Gets the automatic annotator config for the app at the
+    given path.
+
+    Args:
+        app_path (str, optional): The location of the MindMeld app
+        config (dict, optional): A config object to use. This will
+            override the config specified by the app's config.py file.
+            If necessary, this object will be expanded to a fully
+            specified config object.
+
+    Returns:
+        dict: The automatic annotator config.
+    """
+
+    if config:
```

Comment:
is `config` here a useful param?  can we just remove it?

Reply:
Removing config from this function entirely. This function will now only be called from auto_annotator.py if a config is not present.

`self.config = config or get_auto_annotator_config(app_path=app_path)
`